app/lib/zeebe-api.js

```diff
@@ -325,6 +327,8 @@ function getErrorReason(error, endpoint) {
       ? errorReasons.CLUSTER_UNAVAILABLE
       : errorReasons.CONTACT_POINT_UNAVAILABLE
     );
+  } else if (code === 13 && type === endpointTypes.CAMUNDA_CLOUD) {
```

Comment:
**Explanation:** I discovered the reason we were getting `UNKNOWN` when the cluster URL is wrong is because we receive code `13` from the server and didn't account for it, therefore getting the default return value. This is only the case for SaaS. I tested this a number of times with various values to make sure